docs/sections/user_guide/cli/tools/config.rst

```diff
@@ -504,137 +256,59 @@ and YAML file ``update.yaml`` with the following contents:
+The examples that follow use the :json-schema:`JSON Schema<understanding-json-schema/reference>` file ``schema.jsonschema`` with contents:
 
-  Shell redirection via ``|``, ``>``, et al. may also be used to stream output to a file, another process, etc.
+.. literalinclude:: config/schema.jsonschema
+   :language: json
 
-* To read the *config* from ``stdin`` and print validation results to ``stdout``:
+and the YAML file ``values.yaml`` with contents:
 
-  .. code-block:: text
+.. literalinclude:: config/values.yaml
+   :language: yaml
 
-     $ cat values.yaml | uw config validate --schema-file schema.jsonschema
-     [2024-01-03T17:26:29]     INFO 0 UW schema-validation errors found
+* To validate a YAML config against a given JSON schema:
 
-* However, reading the *schema* from ``stdin`` is **not** supported:
+  .. literalinclude:: config/validate-pass.cmd
+     :language: text
+     :emphasize-lines: 1
+  .. literalinclude:: config/validate-pass.out
+     :language: text
 
-  .. code-block:: text
+  :shell-redirection:`Shell redirection<>` may also be used to stream output to a file, another process, etc.
 
-     $ cat schema.jsonschema | uw config validate --input-file values.yaml
-     uw config validate: error: the following arguments are required: --schema-file
+* To read the *config* from ``stdin`` and print validation results to ``stdout``:
 
-* If a config fails validation, differences from the schema will be displayed. For example, with ``recipient: World`` removed from ``values.yaml``:
+  .. literalinclude:: config/validate-pass-stdin.cmd
+     :language: text
+     :emphasize-lines: 1
+  .. literalinclude:: config/validate-pass-stdin.out
+     :language: text
 
-  .. code-block:: text
+* If a config fails validation, differences from the schema will be displayed. For example, ``values-bad.yaml``:
 
-     $ uw config validate --schema-file schema.jsonschema --input-file values.yaml
-     [2024-01-03T17:31:19]    ERROR 1 UW schema-validation error found
-     [2024-01-03T17:31:19]    ERROR 'recipient' is a required property
-     [2024-01-03T17:31:19]    ERROR
-     [2024-01-03T17:31:19]    ERROR Failed validating 'required' in schema['properties']['values']:
-     [2024-01-03T17:31:19]    ERROR     {'additionalProperties': False,
-     [2024-01-03T17:31:19]    ERROR      'properties': {'greeting': {'type': 'string'},
-     [2024-01-03T17:31:19]    ERROR                     'recipient': {'type': 'string'}},
-     [2024-01-03T17:31:19]    ERROR      'required': ['greeting', 'recipient'],
-     [2024-01-03T17:31:19]    ERROR      'type': 'object'}
-     [2024-01-03T17:31:19]    ERROR
-     [2024-01-03T17:31:19]    ERROR On instance['values']:
-     [2024-01-03T17:31:19]    ERROR     {'greeting': 'Hello'}
+  .. literalinclude:: config/values-bad.yaml
+     :language: yaml
+  .. literalinclude:: config/validate-fail.cmd
+     :language: text
+     :emphasize-lines: 1
+  .. literalinclude:: config/validate-fail.out
```

Comment:
Looking at the rendered output, I'm not sure this one failed as we describe here in the text. Maybe I am misreading it?

Reply:
Nice catch, thank you! The command file actually used `values.yaml` instead of `values-bad.yaml`. I'm pushing a fix.